projects/every-voice/src/lib/every-voice.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Inject, Injectable, Optional } from "@angular/core";
+import { EVERY_VOICE_CONFIG } from "./every-voice.token";
+import { EveryVoiceConfig } from "./every-voice.config";
+
+@Injectable()
+export class EveryVoiceService {
+  public enableTTS: boolean;
+  private apiUrl: string;
+
+  constructor(
+    @Optional() @Inject(EVERY_VOICE_CONFIG) config: EveryVoiceConfig
+  ) {
+    this.apiUrl = config?.apiUrl ?? "https://default.api/tts";
+    this.enableTTS = config?.enableTTS ?? true;
```

Comment:
It would be better if the default is **true** only if you have defined an **apiUrl** in your **config**.

```suggestion
    this.enableTTS = (config?.enableTTS !== undefined ? config.enableTTS : (config?.apiUrl?.length > 0));
```

Reply:
great point - thanks @deltork